docs/components/Callout.js

```diff
@@ -3,13 +3,13 @@ export default {
   description: 'This is a customizable callout component.',
   snippet:
         `<ao-callout
-      v-if="showCallout"
-      success
-      dismissible
-      :icon-class="'md-icon__check'"
-      @hideCallout="hideCallout">
-      Callout text goes here
-    </ao-callout>`,
+  v-if="showCallout"
+  success
+  dismissible
+  :icon-class="'md-icon__check'"
+  @hideCallout="hideCallout">
```

Comment:
```suggestion
  @hideCallout="hideCallout"
>
```
our convention is for the `>` to be on the next line